packages/sage/src/commands/generate.ts

```diff
@@ -0,0 +1,77 @@
+import { Command } from 'commander'
+import { readFile, writeFile } from 'node:fs/promises'
+import { logger } from '../core/logger.js'
+import type { Manifest, CommandEntry, ContextEntry, EventConfig, CommandOption } from '@roboplay/robo.js'
+import path from 'node:path'
+
+const command = new Command('generate')
+command.command('doc').description('generates a basic doc file for the project').action(generateDocAction)
+export default command
+
+async function generateDocAction() {
+	try {
+		const manifestPath = path.join(process.cwd(), '.robo', 'manifest.json')
```

Comment:
You can import `loadManifest` from the internal Robo.js files for now and use that for loading. I had already planned on cleaning those up by exposing them via a separate module export anyway.

```typescript
// @ts-expect-error - Internal module
import { loadManifest } from '@roboplay/robo.js/dist/cli/utils/manifest.js'

const manifest: Manifest = await loadManifest()
```